src/Security.md

```diff
@@ -20,7 +20,7 @@ You can enable security declaratively or programmatically, as shown below.
 
 ```xml
 <hazelcast xsi:schemaLocation="http://www.hazelcast.com/schema/config
-    http://www.hazelcast.com/schema/config/hazelcast-config-3.3.xsd"
+    https://hazelcast.com/schema/config/hazelcast-config-3.9.xsd"
```

Comment:
Http version will still redirect you to Https, I didn't see any benefit in not changing it.